src/Main.js

```diff
@@ -1,11 +1,25 @@
 import React from "react";
 import HornedBeast from "./HornedBeast.js";
+import HornForm from "./HornForm.js";
 
 import "./Main.css";
 
 class Main extends React.Component{
+  constructor(props) {
+    super(props);
+    this.state = {
+      beastsToShow: this.props.allData
+    }
+  }
```

Comment:
- While your filter of beast is working quite nicely, we want to review these lines. We are passing props as initial state to this component which can cause some unwanted behaviors in some situations.
- Typically, props should be avoided in the initial state unless props are never updated or the component should not react to their updates. I believe if the parent component does not change this value in state, then we may be ok using this approach...I would grab Ryan to get more info on this when you can 👍